contributing/documentation/guidelines.rst

```diff
@@ -0,0 +1,162 @@
+   - ``https://odoo.com/documentation/user/13.0/sales/products_prices/products/variants.html``
+
+#. Relative:
+
+   - ``../prices.html``
+   - ``variants.html``
+
+The relative links are clearly superior in terms of readability and stability: the references
+survive version updates, folder name changes and file tree restructurations.
+
+.. _contributing/line-length-limit:
+
+Start a new line before the 100th character
+===========================================
+
+In RST, it is possible to break a line without forcing a line break on the rendered HTML. Make use
+of this feature to write **lines of maximum 100 characters**. A line break in a sentence results in
+an additional whitespace in HTML. That means that you do not need to leave a trailing whitespace at
+the end of a line to separate words.
+
+.. tip::
+   You can safely break a line around the separators (``-->``) of ``menuselection`` directives and
+   anywhere in a hyperlink reference. For the ``doc``, ``ref`` and ``download`` directives, this is
+   only true for the label part of the reference.
+
+Example: Line breaks within directive and inline markup
+-------------------------------------------------------
+
+.. code-block:: rst
+
+   To register your seller account in Odoo, navigate to :menuselection:`Sales --> Configuration
+   --> Settings --> Amazon Connector --> Amazon Accounts` and click on **CREATE**. The **Seller
+   ID** can be found under the link **Your Merchant Token**.
+
+Be consistent with indentation
+==============================
+
+Use only spaces (never tabs).
+
+Use as many spaces at the beginning of an indented line as needed to align it with the first
+character of the directive in the line above. This usually implies 3 spaces but you only need 2 for
+bulleted lists.
+
+Example: The first ``:`` is below the ``i`` (3 spaces)
+------------------------------------------------------
+
+.. code-block:: rst
+
+   .. image:: media/example.png
+      :align: center
+      :alt: example
+
+Example: The ``:titlesonly:`` and page references start below the ``t`` (3 spaces)
+----------------------------------------------------------------------------------
+
+.. code-block:: rst
+
+   .. toctree::
+      :titlesonly:
+
+      payables/supplier_bills
+      payables/pay
+
+Example: Continuation lines resume below the ``I``’s of “Invoice” (2 spaces)
+----------------------------------------------------------------------------
+
+.. code-block:: rst
+
+   - Invoice on ordered quantity: invoice the full order as soon as the sales order is confirmed.
+   - Invoice on delivered quantity: invoice on what you delivered even if it's a partial delivery.
+
+.. _contributing/menuselection:
+
+Use the menuselection directive
+===============================
+
+Although chaining characters ``‣`` and menu names works fine to indicate a user which menus to
+click, it is best to use the ``menuselection`` directive (see
```

Comment:
I would add a colon after "see" so it separates it from the doc title.
(see: Spice your writing...)